samples/bluetooth/ble_pwr_profiling/README.rst

```diff
@@ -122,6 +122,9 @@ LED 0:
    This causes a delay between the SoC wake up and button press processing.
    If you want to start advertising on system start, you must keep pressing the button until you see a log message confirming the advertising start on the terminal.
 
+.. note::
+   To observe the LED the :kconfig:option:`CONFIG_BLE_PWR_PROFILING_LED` Kconfig option must be set.
```

Comment:
Can add a line here telling that the current consumption will increase if the LED is turned on.

Reply:
I feel like this sentence is missing something such as `To observe the LED being turned on...`